src/css/themes/light.scss

```diff
@@ -0,0 +1,68 @@
+@mixin theme-light {
+    .title {
+        color: #333333;
+    }
+
+    .bg-primary {
+        background-color: #f1f1f1!important;
+        color: #222;
+        a {
+            color: blue;
+        }
+    }
+
+    .bg-secondary {
+        background-color: #f1f1f1!important;
+        color: #222;
+    }
+
+    .react-grid-layout {
+        background-color: #FFFFFF;
+    }
+
+    body {
+        color: #333333;
+        background-color: #FFFFFF;
+    }
+
+    .close-pane-button {
+        background: none;
+        color: #333333;
+    }
+
+    .card {
+        border: 1px solid #f1f1f1;
+    }
+
+    .sets-manager .ant-tabs .ant-tabs-bar {
+        border-bottom: 1px solid #555;
+        background-color: transparent;
+    }
+
+    .sets-manager .sets-manager-icon-bar i:hover {
+        background-color: silver;
+    }
+
+    .sets-manager .ant-tree.ant-tree-block-node li .ant-tree-node-content-wrapper .ant-tree-title-save-button {
+        background-color: #FFFFFF;
+    }
+
+    .sets-manager .ant-tree.ant-tree-block-node li .ant-tree-node-content-wrapper .ant-tree-title {
+        color: #333333;
+    }
+
+    .sets-manager .ant-tabs.ant-tabs-card .ant-tabs-card-bar .ant-tabs-tab {
+        background-color: rgb(138, 138, 138);
+        border: 1px solid #555;
+    }
+
+    .sets-manager .ant-tabs.ant-tabs-card .ant-tabs-card-bar .ant-tabs-tab .ant-tabs-close-x {
+        color: white;
+    }
+
+    .sets-manager .ant-tabs.ant-tabs-card .ant-tabs-card-bar .ant-tabs-tab.ant-tabs-tab-active {
+        background-color: rgb(138, 138, 138);
+        border: 1px solid #555;
+        color: #000;
+    }
+}
```

Comment:
I'd worry about keeping these two in sync. Could you make this a single [mixin with an argument](https://sass-lang.com/documentation/at-rules/mixin#arguments)?

Reply:
Great idea, I did not know about that feature